qt/Torrent.h

```diff
@@ -590,33 +509,78 @@ class Torrent : public QObject
     static KeyList const mainStatKeys;
 
 private:
-    int getInt(int key) const;
-    bool getBool(int key) const;
-    QIcon getIcon(int key) const;
-    double getDouble(int key) const;
-    qulonglong getSize(int key) const;
-    QString getString(int key) const;
-    time_t getTime(int key) const;
-
-    bool setInt(int key, int value);
-    bool setBool(int key, bool value);
-    bool setIcon(int key, QIcon const&);
-    bool setDouble(int key, double);
-    bool setString(int key, char const*, size_t len);
-    bool setSize(int key, qulonglong);
-    bool setTime(int key, time_t);
+    void updateMimeIcon();
+
+private:
+    int const id_;
+
+    bool downloadLimited_ = {};
+    bool honorsSessionLimits_ = {};
+    bool isFinished_ = {};
+    bool isPrivate_ = {};
+    bool isStalled_ = {};
+    bool uploadLimited_ = {};
+
+    time_t activityDate_ = {};
+    time_t addedDate_ = {};
+    time_t dateCreated_ = {};
+    time_t editDate_ = {};
+    time_t manualAnnounceTime_ = {};
+    time_t startDate_ = {};
+
+    int bandwidthPriority_ = {};
+    int downloadLimit_ = {};
+    int error_ = {};
+    int eta_ = {};
+    int peer_limit_ = {};
```

Comment:
```suggestion
    int peerLimit_ = {};
```

Reply:
Changed in 0aaf236e99b7462885d2c4134b6dbac095cb2366